nfs-ha-cluster-ubuntu/README.md

```diff
@@ -0,0 +1,53 @@
+# Deploy a Highly Available NFS Cluster with Ubuntu VMs
+
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fnfs-ha-cluster-ubuntu%2Fazuredeploy.json)  [![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.png)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fnfs-ha-cluster-ubuntu%2Fazuredeploy.json)
+
+This template allows you to create a highly available NFS cluster on Azure with 2 Ubuntu VMs. The following diagram shows the architecture of the deployed cluster.
+
+![cluster_diagram](images/NFS-HA-Arch.png "Diagram of deployed cluster")
+
+## Usage
+
+### Template Parameters
+
+There are many parameters this template requires. Some notable and important ones are as follows:
+
+- `subnetId`: This is where the highly available NFS cluster will be deployed. You need to give the Azure resource ID of an already existing Azure subnet where the cluster should be deployed. You may find that the template accept `default` as this parameter's value, but that's only for the Azure quickstart templates repo's CI validation purpose (because for CI, we can't provide an already existing vnet, so we have to create one on the fly, and `default` will do just that---however, the IP ranges are fixed to `10.0.0.0/16` for the vnet and `10.0.0.0/24` for the subnet, so you shouldn't really rely on this unless your desired IP ranges match these hard-coded ones).
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#template-pre-requisites for how to use prereqs in CI

Reply:
Thanks for the pointer. Updated the template w/ the suggested GEN-PREREQ practice.